### PR TITLE
new blog entry - Fuse Tooling improvements

### PR DESCRIPTION
**Pull request by lhein**
please wait with merge until approved by AC

### CODE REVIEW

File: blog/2015-02-27-fusetooling-news.adoc [hunk 1]
```diff
@@ -0,0 +1,56 @@
+= Improved JBoss Fuse Tooling coming soon... 
+:page-layout: blog
+:page-author: lhein
+:page-tags: [fuse, camel, jbosstools, devstudio, jbosscentral]
+
+
+I am happy to announce some of the highlights of the upcoming release of the http://tools.jboss.org/features/apachecamel.html[Red Hat JBoss Fuse Tooling].
+It will be available via early access in the JBoss Tools Integration Stack 8. 
+
```
Comment (maxandersen) [inline blog/2015-02-27-fusetooling-news.adoc]: It is JBoss Tools Integration Stack 4.2/Developer Studio Integration Stack 8.0

File: blog/2015-02-27-fusetooling-news.adoc [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+== What is new?
+
+There are a lot of improvements and bugfixes and I just want to pick the most important things for now. You can see a full list of changes in the *What’s New* section for the release.
```
Comment (maxandersen) [inline blog/2015-02-27-fusetooling-news.adoc]: would been nice if IS would make NN's then we could link to them ;)

**Comment by aileenc** [comment]
ok

**Comment by maxandersen** [comment]
merged and pushed!

**Pull request merged**